### PR TITLE
ocserv: split UDP and TCP port configs

### PR DESCRIPTION
**Pull request by billsq**
Maintainer: @nmav 
Compile tested: (mvebu, WRT1900ACS, LEDE 17.01.1)
Run tested: (mvebu, WRT1900ACS, LEDE 17.01.1)

Added a new config entry udp_port to split UDP port from TCP. This is useful when particular port is blocked by the ISP.

udp_port falls back to port if not set to be compatible with current config file.

Also fixed an ifname typo from the last commit.

Signed-off-by: Qian Sheng <billsq@billsq.me>

### CODE REVIEW

File: net/ocserv/files/ocserv.init [hunk 1]
```diff
@@ -48,8 +49,7 @@ setup_config() {
 
 		ping_leases=1
 		local ifname
-		network_get_device ifname lan
-		if test -n "ifname";then
+		if network_get_device ifname lan; then
```
Comment (nmav) [inline net/ocserv/files/ocserv.init]: Is that reliable?

Comment (hnyman) [inline net/ocserv/files/ocserv.init]: @jow- suggested that in https://github.com/openwrt/packages/pull/4301#discussion_r113640821 after you had already merged that previous PR.

**Comment by nmav** [comment]
I guess it is then.

**Comment by nmav** [comment]
thank you

**Pull request merged**